index.js

```diff
@@ -0,0 +1,7 @@
+// code away!
+require('dotenv').config();
+
+const server = require('./api/server.js');
```

Comment:
I was running this locally and it threw an error--did you do some moving around of files? Based on your current structure this should be pointing to ./server/server.js instead